Makefile.core.mk

```diff
@@ -48,12 +48,21 @@ GIT_STATUS := $(shell git diff-index --quiet HEAD -- 2> /dev/null; if [ "$$?" =
 endif
 
 # Linker flags for the go builds
+CGO_ENABLED ?= 0
```

Comment:
I would not assume this CGO_ENABLED default value to 0. Do we need to explicitly set this to 0 in a regular upstream build ? 
```suggestion

```